tests/tensor_search/test_searchable_attributes.py

```diff
@@ -0,0 +1,201 @@
+import os
+from unittest import mock
+
+from marqo.core.models.marqo_index import *
+from marqo.core.models.marqo_index_request import FieldRequest
+from marqo.tensor_search import tensor_search
+from marqo.tensor_search.models.add_docs_objects import AddDocsParams
+from tests.marqo_test import MarqoTestCase
+
+
+class TestSearchStructured(MarqoTestCase):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+
+        default_text_index = cls.structured_marqo_index_request(
```

Comment:
Looks like you only need one of these indexes. I would use a much simpler index as the extra fields aren't relevant for this test. Also I think you should test with an unstructured index too.